src/tools/html-checker/main.rs

```diff
@@ -58,12 +46,21 @@ fn check_html_file(file: &Path) -> usize {
     }
 }
 
+const DOCS_TO_CHECK: &[&str] =
+    &["alloc", "core", "proc_macro", "implementors", "src", "std", "test"];
```

Comment:
Rather than only checking specific crates, could you instead filter out the books you don't want to check?

Reply:
There are A LOT of different and I can't know for sure which one will be added. I initially filtered out but it was a huge mess. We don't generate that many crates in the end so I preferred going this way.